src/test/java/testPackage01/Test_PDF.java

```diff
@@ -0,0 +1,23 @@
+package testPackage01;
+
+import org.testng.annotations.Test;
+
+import com.shaft.cli.FileActions;
+import com.shaft.tools.io.PdfFileManager;
+import com.shaft.tools.io.PdfFileManager.DeleteFileAfterValidationStatus;
+import com.shaft.validation.Assertions;
+
+public class Test_PDF {
+
+	@Test
+	public void testPDF_Downloaded() {
+		String url = FileActions.getAbsolutePath("src/test/resources/TestDataFiles/", "sample.pdf");
+
+		System.out.println(url);
+
+		String content = new PdfFileManager(url, 7000).readPDFContentFromDownloadedPDF(1, 1,
```

Comment:
We can make this method accept relative path and then do an internal call to convert it to absolute path. That would provide easier user experience